docs/docs/administration/settings/signal/filter.mdx

```diff
@@ -0,0 +1,28 @@
+# Filter
+
+Signal Filters are used to define how Signals should be grouped together or when they should be snoozed.
+
+## Snooze Filter
+
+Snooze filters make use of entities extracted from Signals to define when a Signal should be snoozed.
+For example, you can create a Snooze Filter that will snooze all incoming Signals that contain a specific JA3 hash.
+You also have the option to create a Snooze Filter without specifiyng any entities, which will snooze all incoming Signals.
+This feature is useful when an influx of Signals is expected and you want to avoid being overwhelmed by them.
```

Comment:
```suggestion
This feature is useful when an influx of signals is expected for a given period of time (e.g. some known administration activities) and you want to temporarily stop cases from being created. Even when a signal is snoozed it will still be processed and associated entities will be created.
```